### PR TITLE
chapter 1 section 4 exercise problems

### PR DESCRIPTION
**Pull request by DandyDev01**
completed image descriptions for exercise problems

### CODE REVIEW

File: ptx/sec_limit_onesided.ptx [hunk 1]
```diff
@@ -1648,10 +1730,30 @@
                   \end{tikzpicture}
                 </latex-image>
                 <description>
-
+                  <p>
+                    Graph that shows the domain and rage -5 to 5. The graph is
```
Comment (sean-fitzpatrick) [inline ptx/sec_limit_onesided.ptx]: typo: range

File: ptx/sec_limit_onesided.ptx [hunk 2]
```diff
@@ -1648,10 +1730,30 @@
...
+                    made up of eight seperate lines that do not connect to each
```
Comment (sean-fitzpatrick) [inline ptx/sec_limit_onesided.ptx]: separate

File: ptx/sec_limit_onesided.ptx [hunk 3]
```diff
@@ -1648,10 +1730,30 @@
                   \end{tikzpicture}
                 </latex-image>
                 <description>
-
+                  <p>
+                    Graph that shows the domain and rage -5 to 5. The graph is
+                    made up of eight seperate lines that do not connect to each 
+                    other. Each of the lines is parallel with the x-axis, defined
+                    by a solid dot on its left and a hollo dot on its right, and
```
Comment (sean-fitzpatrick) [inline ptx/sec_limit_onesided.ptx]: hollow

**Comment by sean-fitzpatrick** [comment]
Great, thanks! Just a few spelling typos to fix.

**Comment by DandyDev01** [comment]
fixed the spelling

**Pull request merged**